canaries/2D_image_stream.py

```diff
@@ -56,22 +54,23 @@ def get_frames_per_sec(ordered_frames):
         ordered_frames.popleft()
     return len(ordered_frames)
 
-async def detection_stream(robot, llist2):
+
+async def detection_stream(robot):
+    llist = deque()
     transform_cam_name = "transform"
     transform = Camera.from_robot(robot, transform_cam_name)
     logging.info(f"found camera {transform_cam_name}")
     while True:
         # This is to stop this script just before the start of the next hour.
         current_min = int(datetime.datetime.now().strftime("%M"))
         if current_min == 59:
-            await close_robot(robot)
             return
-        detect_img = await transform.get_image(CameraMimeType.JPEG) # default is PNG, JPEG is faster
```

Comment:
This should just use the default which is `JPEG`. This is so our experience (technically the canary's experience) is as close to the user's experience as possible. Since users will likely leave this as default, we've opted to leave out MIME types too.